internal/daemon/worker/auth_rotation_test.go

```diff
@@ -28,6 +28,8 @@ import (
 // TestRotationTicking ensures that we see new credential information for a
 // worker on both the controller side and worker side in a periodic fashion
 func TestRotationTicking(t *testing.T) {
+	t.Parallel()
```

Comment:
Was this intentional?

Reply:
yes, I think this should be able to run in parallel. I can't see any global resources used